etl/steps/data/garden/demography/2023-03-31/population/population.meta.yml

```diff
@@ -1,3 +1,9 @@
+collapsed_origin:
```

Comment:
`collapsed_origin` sound like a Christopher Nolan movie. Shall we call it `origin_combined`? If it's annoying, leave it like that.